scripts/modules/models-b2bcontacts.js

```diff
@@ -0,0 +1,71 @@
+
+define(["modules/api", 'underscore', "modules/backbone-mozu", 'modules/models-address',  "hyprlive", "modules/models-product"], function (api, _, Backbone, AddressModels, Hypr, ProductModels, ReturnModels) {
+
+           
+    var QuoteItem = Backbone.MozuModel.extend({
+        relations: {
+            product: ProductModels.Product
+        },
+        helpers: ['uniqueProductCode'],
+        uniqueProductCode: function () {
+            //Takes into account product variation code
+            var self = this,
+                productCode = self.get('productCode');
+    
+            if (!productCode) {
+                productCode = (self.get('product').get('variationProductCode')) ? self.get('product').get('variationProductCode') : self.get('product').get('productCode');
+            }
+            return productCode;
+        },
+        toJSON: function () {
+            var j = Backbone.MozuModel.prototype.toJSON.apply(this, arguments);
+            if (j.parent) {
+                j.parent = j.parent.toJSON();
+            }
+            return j;
+        }
+    }),
+    
+        QuoteItemsList = Backbone.Collection.extend({
+            model: QuoteItem
+        }),
+    
+        B2bContact = Backbone.MozuModel.extend({
+            mozuType: 'quote',
+            idAttribute: 'id',
+            relations: {
+                items: QuoteItemsList
+            },
+            handlesMessages: true,
+            initialize: function () {
+                var self = this;
+            },
+            toJSON: function () {
+                var self = this,
+                    jsonItems = [];
+                this.get('items').each(function (item) {
+                    jsonItems.push(item.toJSON());
+                });
+                this.set('items', jsonItems);
+                var j = Backbone.MozuModel.prototype.toJSON.apply(this, arguments);
+                return j;
+            }
+        }),
+        B2bContactCollection = Backbone.MozuPagedCollection.extend({
+            mozuType: 'quotes',
```

Comment:
similar, double check you need mozuType 'quotes' here since you are making a collection of B2bContact.